src/App.js

```diff
@@ -1,33 +1,61 @@
 import React from 'react';
-import { Routes, Route } from 'react-router-dom';
-// import { useSelector } from 'react-redux';
+import {
+  Routes, Route, Navigate,
+} from 'react-router-dom';
+import { useSelector } from 'react-redux';
 import Cars from './components/cars/Cars';
 import NavBar from './components/navbar/NavBar';
 import ReserveForm from './components/reservations/ReserveForm';
 import Registration from './components/auth/Registration';
 import Login from './components/auth/Login';
 import MyReservations from './components/reservations/MyReservations';
-import AddCarFrom from './components/reservations/AddCarFrom';
+import AddCarForm from './components/reservations/AddCarForm';
 import DeleteCarForm from './components/reservations/DeleteCarForm';
-
+import CarDetails from './components/cars/CarDetails';
+import MobileNavBar from './components/navbar/MobileNavBar';
+import ProtectedAdminRoutes from './components/auth/ProtectedAdminRoutes';
+import ProtectedRoutes from './components/auth/ProtectedRoutes';
 import './App.css';
+import './media-queries-breakpoints/Small.css';
+import './media-queries-breakpoints/Large.css';
+import './media-queries-breakpoints/XLarge.css';
+import './media-queries-breakpoints/XXLarge.css';
+import './media-queries-breakpoints/Desktop.css';
 
 function App() {
-  // const user = useSelector((state) => state);
-  // console.log(user);
+  // const navigate = useNavigate();
+
+  // const Enroute = () => {
+
+  //   alert(notification);
+  //   navigate('/cars');
+  // }
```

Comment:
- [ ] Kindly, remove this old piece of commented code.

Reply:
Removed. Thanks